text/0000-tool-attributes.md

```diff
@@ -155,6 +181,29 @@ Tool-scoped attributes should be preserved by the compiler for as long as
 possible through compilation. This allows tools which plug into the compiler
 (like Clippy) to observe these attributes on items during type checking, etc.
 
+Likewise, white-listed tools may be used a prefix for lints. So for example,
+`rustfmt::foo` and `clippy::bar` are both valid lint names, from the compiler's
+perspective.
+
+
+### Activation and unused attibutes/lints
+
+For each name on the whitelist, it is indicated if the name is active for
+attributes or lints. A name is only activated if required. So for example,
+`rustdoc` will not be activated at all until it takes advantage of this feature.
+I expect `clippy` will be activated only for lints, and `rustfmt` only for
```

Comment:
`clippy` will be activated for both, since we have some attributes (and want to extend the set of) in order to make some lints configurable.